docs/style-guide.rst

```diff
@@ -439,15 +439,15 @@ Yes:
 
     x = 1;
     y = 2;
-    long_variable = 3;
+    longVariable = 3;
 
 No:
 
 .. code-block:: solidity
 
     x             = 1;
     y             = 2;
-    long_variable = 3;
+    longVariable = 3;
```

Comment:
Some spaces need to be deleted in the first two lines.

Reply:
good catch!